app/src/org/commcare/tasks/DataPullTask.java

```diff
@@ -306,6 +306,8 @@ private static boolean shouldGenerateFirstKey() {
             return processErrorResponseWithMessage(pullResponse);
         } else if (responseCode == 500) {
             return handleServerError();
+        } else if (responseCode == 503) {

```

Comment:
I'm really confused, I thought I added this to a previous release. Did that change not get replicated to master?

Reply:
@ctsims  The change you made was for form submissions and not restore.